src/components/SQFormDialog/SQFormDialog.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {Formik} from 'formik';
-import SQFormDialogInner from './SQFormDialogInner';
+import SQFormDialogInner, {SQFormDialogInnerProps} from './SQFormDialogInner';
```

Comment:
I think our convention is to split this into 2 import statements since of them is a type.

Reply:
agreed